### ISSUE TITLE
EHF & SMPL-X real world coordinate??

### ISSUE THREAD

**Issue by sunwonlikeyou** [vchoutas/expose#22]
I wonder a unit of measure of EHF and SMPL-X .

Are the two expressed in meter??
and also  world coordinate??

**Comment by vchoutas** [comment]
Yes, all of them are in meters. The world coordinate comes from the scanner. The transformation to the camera frame is included in the EHF data, in the EHF_camera.txt file. The camera rotation is in axis-angle format. What would you like to do with it?

**Comment by sunwonlikeyou** [comment]
Thank you :)
I just want to compare other datasets.

**Comment by vchoutas** [comment]
Can I close this issue?

**Comment by booker-max** [comment]
Do you have the training code for this paper? Can you share it with me? This is my email address 1654617276wj@gmail.com，Thank you very much.！
